src/api/get-installations.js

```diff
@@ -59,6 +86,7 @@ async function findWindowsBrowsers () {
     if (edgeAlias)
         installations['edge'] = edgeAlias;
 
+
```

Comment:
Now you change the code page 5 times instead of 2:
[one](https://github.com/DevExpress/testcafe-browser-natives/pull/59/files#diff-d30903c8763835cd105158ddd2fcb731R67), [two](https://github.com/DevExpress/testcafe-browser-natives/pull/59/files#diff-d30903c8763835cd105158ddd2fcb731R71), [three](https://github.com/DevExpress/testcafe-browser-natives/pull/59/files#diff-d30903c8763835cd105158ddd2fcb731R51), [four](https://github.com/DevExpress/testcafe-browser-natives/pull/59/files#diff-d30903c8763835cd105158ddd2fcb731R53), [five](https://github.com/DevExpress/testcafe-browser-natives/pull/59/files#diff-d30903c8763835cd105158ddd2fcb731R55)

But we should have just the following:
```js
findWindowsBrowsers () {
    setCodePage(65001);
    
    //do smthg
    detectMicrosoftEdge();
    //do smthg

    restoreCodePage();
    return ...;
}
```